docs/workshop/intro_to_project.md

```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+## Introduction to AIOps
+
+![Intro to AIOps](../assets/images/intro_to_aiops.png)
+
+AIOps stands for Artificial Intelligence for IT Operations. AIOps is a critical component of supporting any Open Hybrid Cloud infrastructure. In our opinion,  it’s mostly a cultural change, very much as we saw with the devops movement.Here we combined the two cultural mentalities from dev and ops to create a new culture, the devops culture. And the same cultural change is going to happen with AIOps. The devops culture and the data science culture are using different tools and have some different mindsets.
+
+
+## Introduction to AI4CI
+
+![Intro to AI4CI](../assets/images/intro_to_ai4ci.png)
+
+Continuous integration (CI) is the practice of automating the integration of code changes from multiple contributors into a single software project. CI/CD is a solution to the problems that integrating new code can cause for development and operations teams also known as  integration hell.
+
+And, **AI4CI** is AI applied to CI/CD data. It an intelligent open source AIOps toolkit which can be used to better monitor builds  in order to help the development lifecycle.
+
+
+## Introduction to Operate First
+
+![Intro to Operate First](../assets/images/intro_to_opf.png)
+
+Operate First is an initiative to “operate” software in a production-grade environment - bringing users, developers and operators closer together.  It uses the same community-building process of open-source projects, but extended to ops procedures and data. Operate First enables collaboration between open source developers and cloud providers, and AIOps supports this collaboration by creating a new set of tools around CI/CD processes, which could otherwise be a pain point during the development and production of open source projects.
```

Comment:
```suggestion
[Operate First](https://www.operate-first.cloud/) is an initiative to “operate” software in a production-grade environment - bringing users, developers and operators closer together.  It uses the same community-building process of open-source projects, but extended to ops procedures and data. Operate First enables collaboration between open source developers and cloud providers, and AIOps supports this collaboration by creating a new set of tools around CI/CD processes, which could otherwise be a pain point during the development and production of open source projects.
```